### PR TITLE
Update getting_started.rst

### PR DESCRIPTION
**Pull request by hwpang**
### Description

Fixing the path in getting started examples

### Types of changes

- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [x] Documentation updated.

### CODE REVIEW

**Comment by YuanTingHsieh** [comment]
hi @hwpang , thanks for the interest, could you provide the motivation as to why we don't want to "mkdir simulator-example"?

**Comment by hwpang** [comment]
Hi @YuanTingHsieh , with `mkdir simulator-example`, the copied folder has structure `simulator-example/hello-pt/requirements.txt`, while without it, the copied folder has structure `simulator-example/requirements.txt`. I made these changes to make the folder structure under `simulator-example` consistent throughout the getting started example.

File: docs/getting_started.rst [hunk 1]
```diff
@@ -295,7 +295,6 @@ to a working directory:
 
 .. code-block:: shell
 
-  mkdir simulator-example
   cp -rf NVFlare/examples/hello-world/hello-pt simulator-example/
```
Comment (YuanTingHsieh) [inline docs/getting_started.rst:298]: @hwpang to ensure the behavior is always consistent, we probably need to add a `rm -rf simulator-example` in front of this command to achieve what we want

**Review by YuanTingHsieh — COMMENTED** [review]
suggest one change

File: docs/getting_started.rst (hunk 1 above)
Comment (hwpang) [reply docs/getting_started.rst:299]: @YuanTingHsieh That makes sense, thanks! Updated!

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM

@SYangster and @nvkevlu 

other thing I notice we should remove this "python 3.8" requirement text in getting started doc, we can do that in other PR

**Comment by YuanTingHsieh** [comment]
@hwpang 

Can you think of any benefit we change from "mkdir" to "rm -rf"?

**Comment by hwpang** [comment]
@YuanTingHsieh Thanks. I realize that there is a simpler fix than what I implemented before. I have implemented the new fix to add the `hello-pt` in the `python3 -m pip install -r simulator-example/hello-pt/requirements.txt`. This way, we don't need to change any other part of the documentation. I followed through this new instruction locally, and it works for me.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM

**Comment by SYangster** [comment]
Hi @hwpang, it seems that your most recent commit (https://github.com/NVIDIA/NVFlare/pull/2737/commits/4200f77b7b04741468d0082cfe10e36f35d2d71c) was not signed. Once it is signed, then it'll be ready to merge, thanks!

**Comment by hwpang** [comment]
@SYangster Thanks! I have signed and rebased to main.

**Pull request merged**